docs/windows.md

```diff
@@ -320,26 +246,15 @@ Run the following commands to prepare the Node environment needed by antrea-agen
 ```powershell
 mkdir c:\k\antrea
 cd c:\k\antrea
-$TAG="v1.13.0"
+$TAG="v1.14.0"
 curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Clean-AntreaNetwork.ps1
 curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Prepare-ServiceInterface.ps1
 curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Prepare-AntreaAgent.ps1
-.\Prepare-AntreaAgent.ps1
+# use -RunOVSServices $false for containerized OVS!
+.\Prepare-AntreaAgent.ps1 -InstallKubeProxy $false [-RunOVSServices $false]
 ```
 
-The script `Prepare-AntreaAgent.ps1` performs following tasks:
-
-* Prepare network adapter for kube-proxy (for Kubernetes versions prior to 1.26).
-
-    kube-proxy needs a network adapter to configure Kubernetes Services IPs and
-    uses the adapter for proxying connections to Service. Use following script
-    to create the network adapter. The adapter will be deleted automatically by
-    Windows after the Windows Node reboots.
-
-    Starting from Kubernetes 1.26, the kube-proxy userspace mode is no longer
-    supported. For clusters running a version higher than 1.26, you should skip
-    the preparation of the kube-proxy network adapter by executing the command
-    `.\Prepare-AntreaAgent.ps1 -InstallKubeProxy $false`.
+The script `Prepare-AntreaAgent.ps1` performs the following tasks:
```

Comment:
Question: do we have a plan to remove the support for Windows userspace kube-proxy from this document since it is deprecated?

Reply:
I think we should remove it completely from the documentation post v1.15, for the v2.0 release.